lib/gui/app.js

```diff
@@ -188,13 +188,18 @@ app.config(($urlRouterProvider) => {
   $urlRouterProvider.otherwise('/main');
 });
 
-app.config(($provide) => {
+app.config(($provide, $qProvider) => {
+
+  // Prevent unhandled rejections from $uibModalInstance.dismiss() from generating errors
+  $qProvider.errorOnUnhandledRejections(false);
```

Comment:
"Possibly unhandled rejection" errors happen with both `v1.3.2` and `v2.5.0` of `angular-ui-bootstrap` after upgrading to `Angular 1.6`